sf-docker/README.md

```diff
@@ -31,5 +31,7 @@ When using the nvidia docker runtime SmartFace camera processes need gstreamer p
 
 Other services which could use GPU needs also uncomment environment variable `Gpu__GpuEnabled=true`. This is necessary for extractor, detector, pedestrian-detector and liveness service.
 
+For using specific neural networks runtime it is possible to uncomment environment variable `Gpu__NeuralRuntime` which can have values `Default`, `Cuda` or `Tensor`. The GPU needs support these neural runtimes. When using `Tensor` you can uncomment mapping `"/var/tmp/innovatrics/tensor-rt:/var/tmp/innovatrics/tensor-rt"` to retain TensorRT cache files in the host when container is recreated. This can be helpful as generating cache files is longer operation which needs to be performed before the first run of neural network. Setting neural network runtime is possible for camera, extractor, detector, pedestrian-detector and liveness services.
```

Comment:
[Minor]
```suggestion
For using specific neural networks runtime it is possible to uncomment environment variable `Gpu__NeuralRuntime` which can have values `Default`, `Cuda` or `Tensor`. The GPU needs to support these neural runtimes. When using `Tensor` you can uncomment mapping `"/var/tmp/innovatrics/tensor-rt:/var/tmp/innovatrics/tensor-rt"` to retain TensorRT cache files in the host when container is recreated. This can be helpful as generating cache files is longer operation which needs to be performed before the first run of neural network. Setting neural network runtime is possible for camera, extractor, detector, pedestrian-detector and liveness services.
```